src/views/InventoryChannels.vue

```diff
@@ -206,7 +206,11 @@ async function openLinkThresholdFacilitiesToGroupModal(group: any) {
 }
 
 function getFacilityCount(group: any, facilityTypeId: string) {
-  return group.selectedFacilities?.length ? group.selectedFacilities.filter((facility: any) => facility.facilityTypeId === facilityTypeId).length : 0;
+  if(facilityTypeId === 'STORE') {
+    return group.selectedFacilities?.length ? group.selectedFacilities.filter((facility: any) => facility.facilityTypeId === "RETAIL_STORE" || facility.facilityTypeId === "OUTLET_STORE").length : 0;
```

Comment:
Move the check for group.selectedFacilities?.length as a separate check above, as we are checking for the same condition in both the cases. Also moving the same will reduce a check for facilityTypeId.